src/old-link-color.css

```diff
@@ -0,0 +1,4 @@
+.pull-request-title,
+.repository-list--name > a {
+    color: #0052cc !important;
+}
```

Comment:
In what pages does this stylesheet apply? Where can I find markup with this structure: `.repository-list--name > a ` ?

Reply:
The homepage has a sidebar on the right with recently changed repos if you have a lot of repos.